src/utility-bar/bar.test.tsx

```diff
@@ -84,6 +91,46 @@ describe('Shell utility bar', () => {
 		expect(goToFn).not.toHaveBeenCalled();
 	});
 
+	test('should call /logout when isCarbonioCE is true', async () => {
+		jest.spyOn(networkUtils, 'goToLogin').mockImplementation();
+
+		const logoutHandlerMock = jest.fn(logoutHandler);
+		server.use(http.get('/logout', logoutHandlerMock));
+
+		const zxAuthLogoutHandlerMock = jest.fn(zxAuthLogoutHandler);
+		server.use(http.get('/zx/auth/logout', zxAuthLogoutHandlerMock));
+
+		useLoginConfigStore.setState((s) => ({ ...s, isCarbonioCE: true }));
+		const { user } = setup(<ShellUtilityBar />);
+
+		await user.click(screen.getByRoleWithIcon('button', { icon: ICONS.accountUtilityMenu }));
+		await user.click(screen.getByText(/logout/i));
+		await waitForRequest('get', '/logout');
```

Comment:
I think it is safer to invoke the waitForRequest before the setup, and then wait for the returned promise when needed. This because the waitForRequest search for the requests that comes after it has been invoked. Placing the invocation here might be a risk, considering that timings are not always the same across different executions.